Cabal-tests/tests/ParserTests/regressions/Octree-0.5.format

```diff
@@ -1,4 +1,4 @@
-Octree-0.5.cabal:39:3: Non breaking spaces at 39:3, 41:3, 43:3
+Octree-0.5.cabal:39:1: Non breaking spaces at 39:1, 41:1, 43:1
```

Comment:
Are these new positions (column 1) better than the old ones or worse?

Reply:
I think they are slightly better because the indentation token starts at column 1. It was a bit weird to report the position of the next token.

Reply:
Ok, then I book this as "bug fix" for Alex.